doc/api.rst

```diff
@@ -39,28 +35,12 @@ MNE BIDS
    get_head_mri_trans
    get_anonymization_daysback
    print_dir_tree
+   get_entities_from_fname
    get_entity_vals
    get_datatypes
 
-Path
-====
-
-:py:mod:`mne_bids.path`:
-
-.. automodule:: mne_bids.path
-   :no-members:
-   :no-inherited-members:
-
-.. currentmodule:: mne_bids.path
-
-.. autosummary::
-   :toctree: generated/
-
-   BIDSPath
-   get_entities_from_fname
```

Comment:
why are these gone now?

Reply:
Because they're exposed from `mne_bids/__init__.py` (and this is how we import them in all our examples: `from mne_bids import BIDSPath`)

They're still listed in the API docs – just under `mne_bids`, not `mne_bids.path` anymore :)